spacy/pipeline/entity_linker.py

```diff
@@ -212,10 +235,23 @@ def initialize(
         doc_sample = []
         vector_sample = []
         for example in islice(get_examples(), 10):
-            doc_sample.append(example.x)
+            doc = example.x
+            if self.use_gold_ents:
+                doc.ents = example.y.ents
+            doc_sample.append(doc)
             vector_sample.append(self.model.ops.alloc1f(nO))
         assert len(doc_sample) > 0, Errors.E923.format(name=self.name)
         assert len(vector_sample) > 0, Errors.E923.format(name=self.name)
+
+        # XXX This is a hack. Also `has_annotation` doesn't seem to work as expected?
```

Comment:
What is not working exactly with `has_annotation`?

Reply:
I don't remember exactly, so that comment isn't good enough :disappointed: I think that I misunderstood whether it worked on a per-doc or per-pipeline basis.